packages/manager/modules/pci/src/projects/project/data-processing/notebooks/add-notebook/add-notebook.html

```diff
@@ -0,0 +1,254 @@
+                            <div class="d-flex align-items-center mt-2">
+                                <div class="mr-2">
+                                    <a
+                                        href=""
+                                        data-ng-click="$ctrl.scrollTo('privacy')"
+                                        data-translate="data_processing_add_notebook_spec_privacy"
+                                    ></a>
+                                </div>
+                            </div>
+                            <span data-ng-bind="$ctrl.state.privacy"></span>
+                            <!-- Sizing -->
+                            <div class="d-flex align-items-center mt-2">
+                                <div class="mr-2">
+                                    <a
+                                        href=""
+                                        data-ng-click="$ctrl.scrollTo('sizing')"
+                                        data-translate="data_processing_add_notebook_spec_sizing"
+                                    ></a>
+                                </div>
+                            </div>
+                            <div class="d-flex flex-column mb-2 pl-2">
+                                <div class="d-flex">
+                                    <span
+                                        class="font-weight-bold"
+                                        data-translate="data_processing_add_notebook_spec_sizing_notebook"
+                                    ></span>
+                                    <span
+                                        data-ng-bind="$ctrl.state.notebookSizing.notebook"
+                                    ></span>
+                                </div>
+                            </div>
+
+                            <div class="d-flex flex-column mb-2 pl-2">
+                                <div class="d-flex">
+                                    <span
+                                        class="font-weight-bold"
+                                        data-translate="data_processing_add_notebook_spec_sizing_cluster"
+                                    ></span>
+                                    <span
+                                        data-ng-bind="$ctrl.state.notebookSizing.cluster"
+                                    ></span>
+                                </div>
+                            </div>
+                        </oui-tile-description>
+                    </oui-tile-definition>
+                    <oui-tile-definition>
+                        <oui-tile-description>
+                            <div class="break-word">
+                                <a
+                                    href=""
+                                    data-ng-click="$ctrl.scrollTo('prices')"
+                                    data-translate="data_processing_add_notebook_spec_price"
+                                ></a>
```

Comment:
Please use a button instead of an empty href link

Reply:
In this case, we want the anchor to be used as a "scroll to". I don't think a button would be the best UX choice here as this function is usually presented to the user with a link. I tried a ui-button with a link variant but it still shows as a button when hovered. 
Besides, we want to have the same look and feel than we have in the order funnel of the data processing jobs and cloud databases and they both use an empty link.